CHANGELOG.md

```diff
@@ -11,10 +11,24 @@ created.
 ## Upcoming
 
 ### Added
+- There are new commandline options for controlling PyTorch Lightning training: 
+  - `max_num_gpus` controls how many GPUs are used at most for training (default: all GPUs, value -1).
+  - `generate_report` controls if a HTML report will be written (default: True) 
+  - `recovery_checkpoint_save_interval` determines how often a checkpoint for training recovery is saved.
 - New extensions of SegmentationModelBases `HeadAndNeckBase` and `ProstateBase`. Use these classes to build your own Head&Neck or Prostate models, by just providing a list of foreground classes.
 - Grouped dataset splits and k-fold cross-validation. This allows, for example, training on datasets with multiple images per subject without leaking data from the same subject across train/test/validation sets or cross-validation folds. To use this functionality, simply provide the name of the CSV grouping column (`group_column`) when creating the `DatasetSplits` object in your model config's `get_model_train_test_dataset_splits()` method. See the `InnerEye.ML.utils.split_dataset.DatasetSplits` class for details.
 
 ### Changed
+
+- The codebase has undergone a massive refactoring, to use PyTorch Lightning as the foundation for all training. As
```

Comment:
We should add a note about Pytorch Lightning and about DDP capability in the README as well.